packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -90,7 +92,7 @@ export interface GraphQLTransformOptions {
   readonly overrideConfig?: OverrideConfig;
 }
 export type StackMapping = { [resourceId: string]: string };
-export class GraphQLTransform {
+export class GraphQLTransform { 
```

Comment:
I'm not a fan of committing linting changes alongside actual code changes. I would remove the linting updates from this PR.

Reply:
sure, can do that!

Reply:
Thanks @manaswi223!